tools/releases/helm.sh

```diff
@@ -69,6 +69,22 @@ function package {
       --package-path "${CHARTS_PACKAGE_PATH}" \
       "${dir}"
 
+    # repackage archive to remove potential duplicates
+    for f in "${CHARTS_PACKAGE_PATH}"/*.tgz; do
+      local tmpdir
+      tmpdir=$(mktemp --directory)
+
+      tar -xzf "${f}" --directory "${tmpdir}"
+
+      tar -czvf "${f}" \
+        --directory "${tmpdir}" \
+        --owner 0 \
+        --group 0 \
+        --no-same-owner \
```

Comment:
What do these do? According to the man page they only have an effect when extracting

Reply:
You are right. I removed them.